Taskfile.yml

```diff
@@ -268,6 +268,31 @@ tasks:
         -compat-url='mongodb://username:password@127.0.0.1:47018/?tls=true&tlsCertificateKeyFile=../build/certs/client.pem&tlsCaFile=../build/certs/rootCA-cert.pem'
         -disable-filter-pushdown={{.DISABLE_FILTER_PUSHDOWN}}
 
+  test-integration-mysql:
```

Comment:
Let's always use the same order: PostgreSQL, SQLite, MySQL, HANA, MongoDB. So this task should be above `test-integration-hana`